WeakAuras/GenericTrigger.lua

```diff
@@ -3434,6 +3435,14 @@ function WeakAuras.WatchUnitChange(unit)
       end
     end
 
+    local function markerInit(unit)
+      watchUnitChange.raidmark[unit] = GetRaidTargetIndex(unit) or 0
+    end
+
+    local function markerClear(unit)
+      watchUnitChange.raidmark[unit] = nil
+    end
```

Comment:
For almost every event, e.g. PLAYER_TARGET_CHANGED, we don't need to actually check whether the mark has changed, because we if the guid is different we'll send a UNIT_CHANGED anyway, and if it isn't different (e.g. a bogus event) then presumably the mark hasn't either.